qa/mixed-cluster/src/test/java/org/elasticsearch/backwards/IndexingIT.java

```diff
@@ -283,87 +275,6 @@ public void testUpdateSnapshotStatus() throws Exception {
         request.setJsonEntity("{\"indices\": \"" + index + "\"}");
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/79930")
-    public void testSyncedFlushTransition() throws Exception {
-        Nodes nodes = buildNodeAndVersions();
-        assertTrue("bwc version is on 7.x", nodes.getBWCVersion().before(Version.V_8_0_0));
```

Comment:
I would think this test still makes sense and we should instead use `assumeTrue` here? The transition code is still here and should be until we decide to break it completely, i.e., not support mixed clusters to 7.x.